helm/raster/config/mappings.py

```diff
@@ -39,7 +39,9 @@
         'pycsw:TempExtent_begin': 'source_start_date',
         'pycsw:TempExtent_end': 'source_end_date',
         'pycsw:Resolution': 'max_resolution_deg',
+        'pycsw:minResolutionDeg': 'min_resolution_deg',
```

Comment:
please update the following files:
https://github.com/MapColonies/pycsw/blob/762f18cbe0eddef135d384d0c9e844944eb145fd/pycsw/plugins/profiles/mc_raster/mc_raster.py#L40

https://github.com/MapColonies/pycsw/blob/762f18cbe0eddef135d384d0c9e844944eb145fd/pycsw/plugins/profiles/mc_raster/schemas/mcraster-record.xsd#L68